### ISSUE TITLE
The license_file parameter is deprecated, use license_files instead.

### ISSUE THREAD

**Issue by kplgrwl** [Snowflake-Labs/schemachange#183]
We recently started getting this error on our Schemachange (version 3.5.2). Is there any workaround to fix it?

![image](https://github.com/Snowflake-Labs/schemachange/assets/78981499/97e49810-1e53-4a1b-8c5d-ea2dae8241a7)

**Comment by opodoprigora** [comment]
Same issue, running `schemachange: pip install -q schemachange==3.5.2`

**Comment by kplgrwl** [comment]
We were using 3.4.1 in our production and this start showing up on our CI/CD pipeline since last 7/14 (Friday)

**Comment by opodoprigora** [comment]
@kplgrwl same but started today, this is our workflow 

```yml
      - name: Checkout repo
        uses: actions/checkout@v3
        with:
          fetch-depth: 0

      - name: Use Python 3.10
        uses: actions/setup-python@v4
        with:
          python-version: '3.10'

      - name: Install Python Packages
        run: |
          echo "pip install pytest==7.1.3"
          pip install pytest==7.1.3

          echo "Installing Snowflake Python Connector: pip install -q snowflake-connector-python==2.8.0"
          pip install snowflake-connector-python==2.8.0

          echo "Installing schemachange: pip install -q schemachange==3.4.1"
          pip install -q schemachange==3.4.1

          echo "Installing loguru: pip install loguru==0.6.0"
          pip install loguru==0.6.0

          echo "Installing pandas: pip install pandas==1.5.0"
          pip install pandas==1.5.0

          echo "Setting temporary directory to store data by connector"
          export TMPDIR=/tmp
```

**Comment by mjsqu** [comment]
Caused by this issue with Cython3 in pyyaml?
https://github.com/yaml/pyyaml/issues/724

👀

**Comment by ogawade** [comment]
Same issue

**Comment by mjsqu** [comment]
These changes will work-around the issue by updating PyYAML to 6.0 in the dependencies:
https://github.com/mjsqu/schemachange/commit/41b3b5cad472799292d17dbf895650b1e73e8f36

**Comment by vikas-iquanti** [comment]
Same issue. The PyYAML dependency needs to be upgraded to 6.0

**Comment by ogawade** [comment]
Can someone who is collaborator  add pyyaml = 6.0  and merge  it into schemachange master?

**Comment by snk** [comment]
Same issue

**Comment by opodoprigora** [comment]
Thanks @mjsqu , using forked method as an interim solution.

**Comment by sfc-gh-jhansen** [comment]
Included in the new 3.5.3 release. Thanks!
